code/modules/medical/cloning.dm

```diff
@@ -30,6 +30,9 @@
 
 	l_color = "#7BF9FF"
 
+/obj/machinery/cloning/clonepod/empty
```

Comment:
I don't think this is the right order to go about it.
The map version should start with biomass, but the machine itself should start with nothing.